#include <iostream>

int main()
{
    std::cout << "successful run failing test\n";
    return 42;
}
